mne/viz/evoked.py

```diff
@@ -1947,6 +1947,23 @@ def _title_helper_pce(title, picked_types, picks, ch_names, combine):
     return title
 
 
+def _ascii_minus_to_unicode(s):
+    """Replace ASCII-encoded "minus-hyphen" characters with Unicode minus.
+
+    Aux function for ``plot_compare_evokeds`` to prettify ``Evoked.comment``.
+    """
+    if s is None:
+        return None
```

Comment:
(nitpick) a bare `return` will return `None` so this is equivalent.

```suggestion
        return
```